andino_gazebo/urdf/andino.gazebo.xacro

```diff
@@ -44,6 +47,15 @@
       <plugin filename="libgazebo_ros2_control.so" name="gazebo_ros2_control">
           <parameters>$(find andino_control)/config/andino_controllers.yaml</parameters>
       </plugin>
+      <plugin name="andino_joint_state_caster" filename="libgazebo_ros_joint_state_publisher.so">
+      <ros>
+        <remapping>~/out:=joint_states</remapping>
+        <remapping>/tf:=tf</remapping>
+      </ros>
+      <update_rate>10</update_rate>
+      <joint_name>caster_wheel_joint</joint_name>
+      <!-- <joint_name>caster_rotation_joint</joint_name> -->
```

Comment:
I see that you are not using the continuouts joint.
Does it make sense to have an argument in the launch file that you can pass down to the xacro for using or not using a fixed/continuous caster joint?